apiron/endpoint/__init__.py

```diff
@@ -0,0 +1,11 @@
+from apiron.endpoint.endpoint import Endpoint
+from apiron.endpoint.json_endpoint import JsonEndpoint
+from apiron.endpoint.streaming_endpoint import StreamingEndpoint
+from apiron.endpoint.stub_endpoint import StubEndpoint
+
+__all__ = [
```

Comment:
My understanding is that `__all__` should be a list of strings that _represent_ modules, classes, etc. rather than the classes/modules themselves

Reply:
@daneah fixed. Yeah, it needs strings. Nice catch.